package.json

```diff
@@ -12,9 +12,10 @@
     "codecov": "codecov -f coverage/coverage-final.json && codecov -f coverage/jest/coverage-final.json",
     "test:unit": "jest",
     "test:e2e": "npx cypress run",
-    "test": "npm run test:unit && npm run test:e2e"
+    "test": "mkdir -p coverage/jest && npm run test:unit > coverage/jest/coverage.txt && npm run test:e2e"
```

Comment:
@ivelin 

1. Jest does **not** output a `.txt` file for text coverage, rather it outputs it to the console for the developer to see. It can however output a `.json` file ( if you would prefer this ). This was the reason why I used the `>` hack. See [docs](https://jestjs.io/docs/configuration#coveragereporters-arraystring--string-options)

Reply:
I have implemented 2 & 3. I didn't use the `merge-cypress-jest-coverage` package. I wrote some bash commands to move the file around.

Reply:
@vickywane can you share link to the code that solved 1,2 &3?